pkg/services/services.go

```diff
@@ -67,13 +88,8 @@ func kubeProxyConfiguration(debug bool) servicescm.Service {
 		"--hostname-override=NODE_NAME --kubeconfig=%s --cluster-cidr=NODE_SUBNET --log-dir=%s --logtostderr=false "+
 		"--network-name=%s --source-vip=ENDPOINT_IP --enable-dsr=false", windows.KubeProxyPath, windows.KubeconfigPath,
 		windows.KubeProxyLogDir, windows.OVNKubeOverlayNetwork)
-	// Set log level. See doc link for explanation of log levels:
-	// https://docs.openshift.com/container-platform/latest/rest_api/editing-kubelet-log-level-verbosity.html#log-verbosity-descriptions_editing-kubelet-log-level-verbosity
-	if debug {
-		cmd = fmt.Sprintf("%s --v=4", cmd)
-	} else {
-		cmd = fmt.Sprintf("%s --v=2", cmd)
-	}
+	// Set log level
+	cmd = fmt.Sprintf("%s %s", cmd, klogVerbosityArg(debug))
```

Comment:
Thanks for modularizing this but my question still remains. Why do you have to add these args in two places?